theme/site/globals/site.variables

```diff
@@ -12,14 +12,14 @@
      Fonts
 --------------------*/
 
-@headerFont        : "Share Tech Mono", Monaco, Menlo, "Ubuntu Mono", Consolas, source-code-pro, monospace;
-@pageFont          : "Share Tech Mono", Monaco, Menlo, "Ubuntu Mono", Consolas, source-code-pro, monospace;
+@headerFont        : 'Segoe UI', 'Segoe UI Web (West European)', -apple-system, BlinkMacSystemFont, Roboto, 'Helvetica Neue', sans-serif;
```

Comment:
Is the kerning/tracking with the colors and this font accessible? I find some of the words in the different colors hard to read.

Reply:
@srietkerk this is the font msft uses for everything, so i'm sure it meets all of the accessibility standards

Reply:
but you are correct that some of our blocks colors do not

Reply:
Doesn't the accessibility of a font change based on the color and weight you give the font? In particular, I find the weight and the closeness of the letters of the music category to be particularly difficult.
![image](https://github.com/user-attachments/assets/218f2805-146a-46f7-936b-59b81e9457b2)
versus the spacing it was given before
![image](https://github.com/user-attachments/assets/f0cf5b04-1f05-4ba2-9a21-893842daf99b)

Reply:
I think you might be right that it's just the color contrast that makes it harder to read.